protostar/protostar_cli.py

```diff
@@ -24,6 +24,38 @@
     log_color_provider,
 )
 
+PROFILE_ARG = Command.Argument(
+    name="profile",
+    short_name="p",
+    type="str",
+    description="\n".join(
+        [
+            "Specifies active profile configuration.",
+            "#### CI configuration",
+            '```toml title="protostar.toml"',
+            "[protostar.shared_command_configs.ci]",
+            "no_color=true",
+            "```",
+            "`protostar -p ci test`",
+            "",
+            "#### Deployment configuration",
+            '```toml title="protostar.toml"',
+            "[protostar.deploy.devnet]",
+            'gateway_url="http://127.0.0.1:5050/"',
+            "```",
+            "`protostar -p devnet deploy ...`" "",
+        ]
+    ),
+)
+
+
+class ConfigurationProfileCLI(CLIApp):
```

Comment:
CLI is probably a bad name for this abstraction, right now, but I don't have a better idea

Reply:
Maybe we should split responsibilities here in the future somewhat

Reply:
We could, but it's not important right now. It doesn't add any value to the end-user and we have worse code in the repo.

Reply:
Agreed